daphne/http_protocol.py

```diff
@@ -198,6 +199,15 @@ def process(self):
                     if not more_body:
                         break
 
+                # Count completed requests and check against Max
+                self.server._complete_requests_counted += 1
+                if self.server._complete_requests_counted > self.server.max_requests:
+                    logger.info('Max requests completed. Shutting down daphne...')
+                    os.system("ps aux | grep 'daphne'")
+                    bash_command = "kill $(ps aux | grep 'daphne' | awk '{print $2}')"
+                    os.system(bash_command)
+                    sys.exit(0)
+
```

Comment:
Hmmm. OK, so... the expectation would be that we **restart** after X requests, not just shut down right?